packages/core/src/Site/index.ts

```diff
@@ -1611,6 +1678,21 @@ class Site {
     const time = new Date().toLocaleTimeString(this.siteConfig.locale, options);
     this.variableProcessor.addUserDefinedVariableForAllSites('timestamp', time);
   }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars, class-methods-use-this
+  async rebuildPagesBeingViewed(_currentPageViewed: string) {
+    throw new Error('Method not implemented.');
```

Comment:
Are these added for the sake of the type? The message here is a little confusing to understand

Reply:
Yup for types - reasoning [here](https://github.com/MarkBind/markbind/pull/2270#discussion_r1164425215)

Reply:
I see, will need to open an issue to KIV this. if not we could use a more suitable message, such as "Method implemented via prototype".